libvcx/src/api/connection.rs

```diff
@@ -1200,6 +1201,89 @@ pub extern fn vcx_connection_get_their_pw_did(command_handle: u32,
     error::SUCCESS.code_num
 }
 
+#[no_mangle]
+pub extern fn vcx_connection_download_messages(command_handle: CommandHandle,
```

Comment:
I think I'd prefer to have this in `util.rs` named `vcx_messages_download_v2`, following our previous way of introducing enhanced versions of existing functions.